product/views/ManageIQ_Providers_InfraManager.yaml

```diff
@@ -30,13 +30,17 @@ cols:
 - total_storages
 - total_vms
 - total_miq_templates
+- tenant.name
```

Comment:
The field should be either in `cols` or under `include`, not both.

(so, just include, col_order and headers for this one)
(same in the other yaml)